File: 04_File_reader_opgave/Node.js/me.js
export class Me {
  constructor(name, age, hobbies) {
    this.name = name;
    this.age = age;
    this.hobbies = hobbies;
  }
  writeToConsole() {
    console.log(`Name: ${this.name}`);
    console.log(`Age: ${this.age}`);
    console.log(`Hobbies: ${this.hobbies}`);
  }
}

File: 00._Opgaver/03a._Data_parsing_server/C#/03a._Data_parsing_server_project/Me.cs
using System.Text.Json.Serialization;
using System.Xml.Serialization;

[XmlRoot(ElementName = "me")]
public class Me
{
    [XmlElement(ElementName = "name")]
    [JsonPropertyName("name")]
    public string? Name { get; set; }

    [XmlElement(ElementName = "age")]
    [JsonPropertyName("age")]
    public int? Age { get; set; }

    [XmlArray(ElementName = "hobbies")]
    [XmlArrayItem(ElementName = "hobby")]
    [JsonPropertyName("hobbies")]
    public List<string>? Hobbies { get; set; }

    public void WriteToConsole()
    {
        Console.WriteLine($"Name: {Name}");
        Console.WriteLine($"Age: {Age}");
        Console.WriteLine($"Hobbies: {string.Join(", ", Hobbies!)}\n");
    }
}

public class MeResponse
{
    [JsonPropertyName("me")]
    public Me? Me { get; set; }
}